NOTICE

```diff
@@ -1 +1,9 @@
-Copyright (c) 2015 SAP SE or an SAP affiliate company. All rights reserved.
+Copyright (c) 2018-Present CloudFoundry.org Foundation, Inc. All Rights Reserved.
+This project contains software that is Copyright (c) 2015-2018 SAP SE or an SAP affiliate company.

```

Comment:
Should we keep the SAP reference here?

Reply:
I copied that and included it in line 2 (2015-2018).  That should be sufficient, yes?